### PR TITLE
Use streams3

### PR DESCRIPTION
**Pull request by zertosh**
Waiting on https://github.com/deoxxa/duplexer2/pull/12

### CODE REVIEW

**Comment by shinnn** [comment]
duplexer2 v0.1.0 ([that uses Stream3](https://github.com/deoxxa/duplexer2/blob/af38f7b1229d53e343bc65a13c1fc780ea5c9178/package.json#L24)) is released.

**Comment by zertosh** [comment]
Thanks @shinnn!

**Comment by shinnn** [comment]
https://github.com/substack/stream-combiner2/commit/f738fb5b71c21717ccef8938f0c2c46a4a61a527#diff-04c6e90faac2675aa89e2176d2eec7d8R5

> for streams2

After this is merged (or in this PR?), that line also should be updated.

File: package.json [hunk 1]
```diff
@@ -8,11 +8,12 @@
   },
   "dependencies": {
     "duplexer2": "~0.0.2",
-    "through2": "~0.5.1"
+    "readable-stream": "^2.0.2"
   },
   "devDependencies": {
     "tape": "~2.3.0",
     "through": "~2.3.4",
```
Comment (shinnn) [inline package.json]: Not directly related to this PR, but `through` doesn't seem to be used in this project anymore.

**Comment by feross** [comment]
@substack Looks like https://github.com/deoxxa/duplexer2/pull/12 was released, can this PR be merged now?

**Comment by zertosh** [comment]
@feross after this gets merged then there's just module-deps and browserify itself

**Comment by feross** [comment]
@zertosh yeah! we're very close to being completely streams3 in browserify.

**Comment by shinnn** [comment]
LGTM

**Pull request merged**